task/codesigning.ts

```diff
@@ -27,6 +31,7 @@ async function run(): Promise<void> {
 
     let signCertPassword: string = tl.getInput("signCertPassword", true);
     let timeServer: string = tl.getInput("timeServer", true);
+    let enableDebugSwitch: boolean = tl.getBoolInput("enableDebugSwitch", true);
```

Comment:
IMO this shouldn´t be required. Default value should be false. This should also fix the tests :)

`let enableDebugSwitch: boolean = tl.getBoolInput("enableDebugSwitch", false);`